Lib/test/test_email/test_parser.py

```diff
@@ -105,6 +105,12 @@ def message_from_binary_file(s, *args, **kw):
 class TestBytesParser(TestParserBase, TestEmailBase):
     parsers = (message_from_bytes, message_from_binary_file)
 
+class TestFeedParser(TestCustomMessage):
```

Comment:
I don't see a need for this to inherit from `TestCustomMessage`, let's just do `TestEmailBase` instead?

Reply:
Changed to `TestEmailBase`